src/command_base.ts

```diff
@@ -62,9 +62,9 @@ export abstract class BaseCommand {
 	 *
 	 * It is static so it can be used without construction.
 	 */
-	public static help(): string {
+	static help(): string {
```

Comment:
I assume this also indicates `static` is implicitly `public`. Probably Java syntax sneaking in here.

Reply:
Yes; in fact, I think in general, fields are public by default.